launch/wheelchair.launch.py

```diff
@@ -0,0 +1,24 @@
+from launch import LaunchDescription
+from launch.actions import IncludeLaunchDescription, ExecuteProcess
```

Comment:
For the PR as a whole "fixup" the commits to a single one please

Reply:
Also add a .md file to the docs folder explaining how to can be ran this is the file that makes it to the site